float64[6] after_pack_pose
